### ISSUE TITLE
Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value

### ISSUE THREAD

**Issue by bardram** [builttoroam/flutter_plugins#126]
When trying to receive calendar info on **iOS**, I get an 

```
Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value
```

in `SwiftDeviceCalendarPlugin.swift` line **217**. 

This is in the 

```
private func createEventFromEkEvent(calendarId: String, ekEvent: EKEvent) 
```

function, when accessing the `ekEvent.title` value.

**Comment by nickrandolph** [comment]
Hi @bardram are you able to replicate this using the example app that's part of the source code? If so, would you be able to submit a WIP pull request that demonstrates the issue - we can take a look and see if we can reproduce and fix the issue.

**Comment by Brett09** [comment]
Creating an event from Android calendar app (or account's software such as Outlook) and reading that in the plugin from an iOS device will cause this issue, as `ekEvent.title` cannot be null. (Nullable for Android)

The fix will be adding a null check and default to something like "New Event" for both platforms to be consistent. I'll fix this up in the next update.

**Comment by Brett09** [comment]
This has been fixed in v3.1.0
